### DISCUSSION TITLE
I need help with this UserWarning: resource_tracker: There appear to be 1 leaked semaphore objects to clean up at shutdown warnings.warn('resource_tracker: There appear to be %d '

### DISCUSSION THREAD

**Discussion by drogu** [lllyasviel/Fooocus#2690] [Q&A] [answered]
Requested to load SDXL
Loading 1 new model
zsh: killed     python entry_with_update.py
/opt/anaconda3/envs/fooocus/lib/python3.10/multiprocessing/resource_tracker.py:224: UserWarning: resource_tracker: There appear to be 1 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d ' what does this mean How can i solve it

**Comment by mashb1t** [comment, marked as answer]
There seems to be no RAM left on your device. 
=> Make sure you have sufficient RAM available to run Fooocus. See [minimal requirements](https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#minimal-requirement)
